app/models/feeds/megaphone_feed.rb

```diff
@@ -75,4 +75,11 @@ def publish_to_megaphone?
   def mark_as_not_delivered!(episode)
     episode.episode_delivery_status(:megaphone, true).mark_as_not_delivered!
   end
+
+  def advertising_tag_options
+    return [] unless persisted? && config.valid?
+    Megaphone::OrganizationTag.list_by_feed(self).map do |tag|
```

Comment:
Ah, okay - so these exist under the `organization_id` in Megaphone, so we can list them as options rather than manually entering them.  +1.